contracts/F3Parameters.sol

```diff
@@ -62,9 +64,9 @@ contract F3Parameters is Ownable {
     uint128 constant BLOCK_TIME = 30 seconds;
 
     /// @dev The minimum headroom time in seconds before activation.
-    uint128 constant MIN_ACTIVATION_HEADROOM = 3 days;
-	/// @dev The minimum headroom time in blocks.
-	uint128 public constant MIN_ACTIVATION_HEADROOM_BLOCKS = MIN_ACTIVATION_HEADROOM / BLOCK_TIME;
+    uint128 constant MIN_ACTIVATION_HEADROOM = 4 days;
+    /// @dev The minimum headroom time in blocks.
```

Comment:
Apologies if I screwed up the whitespace here.  I thought I have everything set to use spaces insted of tabs, but let me know.

Reply:
It might have been me, no idea.